IFC4x3/Properties/n/NominalWeight_1NkoWVgmHA5xrENqUppTds/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NominalWeight_1NkoWVgmHA5xrENqUppTds" Name="NominalWeight" UniqueId="57bb281f-ab04-4a17-bd4e-5f47b3cdd9f6" PrimaryDataType="IfcMassMeasure" />
```

Comment:
Another PR introduced this. Should be re-used.

Reply:
No merge yet for this property. Will resolve in the second merge for it.